index.html

```diff
@@ -24,14 +24,80 @@
+                </div>
+                <div class="quiz-text">
+                    <span>Featured Category</span>
+                    <h1 class="text-primary">Flowers</h1>
+                    <p>Gardening can be considered both as an art, concerned with arranging plants harmoniously in their surroundings, and as a science, encompassing the principles and techniques of plant cultivation.</p>
+                    <p>Gardening may be a fun and relaxing way to get in touch with nature, but it also has plenty of health benefits and environmental impact.</p>
+                    <p>This quiz test your knowledge around different aspects of gardening and benefits.</p>
+                    <div class="info">
+                        <span><i class="far fa-clock"></i>20 minutes</span>
+                        <span><i class="fas fa-crosshairs"></i> 80 marks</span>
+                    </div>
+                    <button class="btn btn--primary btn--md">Check quizzes</button>
+                </div>
+            </section>
+            <section class="left-expand">
+                <div class="quiz-text">
+                    <span>Featured Category</span>
+                    <h1 class="text-primary">Forest</h1>
+                    <p>Gardening can be considered both as an art, concerned with arranging plants harmoniously in their surroundings, and as a science, encompassing the principles and techniques of plant cultivation.</p>
+                    <p>Gardening may be a fun and relaxing way to get in touch with nature, but it also has plenty of health benefits and environmental impact.</p>
+                    <p>This quiz test your knowledge around different aspects of gardening and benefits.</p>
+                    <div class="info">
+                        <span><i class="far fa-clock"></i>20 minutes</span>
+                        <span><i class="fas fa-crosshairs"></i> 80 marks</span>
+                    </div>
+                    <button class="btn btn--primary btn--md">Check quizzes</button>
+                </div>
+                <div class="image">
+                    <img src="/images/Forest-bro.svg" class="img-fluid" alt="" />
                 </div>
             </section>
         </main>
+        <footer class="footer">
+            <div class="footer_wrapper">
+                <span>Made with <i class="fas fa-heart"></i> by <a href="https://portfolio-alpha-liart.vercel.app/" class="text-gray"> Animesh Garai</a></span>
+                <div class="footer__links">
+                    <a href="https://github.com/animeshgarai09"><i class="fab fa-github"></i></a>
+                    <a href="https://twitter.com/animeshgarai09"><i class="fab fa-twitter"></i></a>
+                    <a href="https://www.linkedin.com/in/animesh-garai-29a5251b4"><i class="fab fa-linkedin"></i></a>
+                    <a href="mailto:animeshgarai09@gmail.com"><i class="fas fa-envelope"></i></a>
```

Comment:
Suggestion- You can add `target="_blank"` to the anchor tag. It will open your social media in a new tab which is a good user experience.

Reply:
Sure will do👍🏻